ui.frontend/src/main/webpack/site/js/buildSearch.ts

```diff
@@ -20,7 +20,27 @@ export const buildSearch = async (
     return
   }
 
-  const { callbacks, autoSuggestionDebounceTime = 500 } = options || {}
+  const { trackingUrl } = searchConfig
+
+  const { callbacks = {}, autoSuggestionDebounceTime = 500 } = options || {}
+
+  if (trackingUrl) {
+    callbacks.onSearchItemClick = (url: string) => {
+      const headers = new Headers()
+      headers.append('Content-Type', 'application/x-www-form-urlencoded')
+      fetch(`${window.location.origin}${trackingUrl}`, {
```

Comment:
why window.location.origin ? can't we just send a relative url?

Reply:
**Thought**

silly question, don't we want to track what item (via an ID or something) got clicked on rather than a generic tracking Url?

Reply:
mmh as I understand this is done in the body:
`body: `trackedUrl=${url}`,`

Reply:
ahhhhhhhhhhhhhhhhhhhhhhhhh my bad, mixed up 'trackingUrl' and 'trackedUrl' :-).
NVM then!